www/src/templates/template-starter-page.js

```diff
@@ -46,21 +46,16 @@ class StarterTemplate extends React.Component {
       !this.state.showAllDeps && allDeps.length - shownDeps.length > 0
 
     // plug for now
-    const isModal = false
```

Comment:
Since this isn't being used, I'm deleting it. If we ever wanted a modal for this, it would probably be something like creating a `<ModalLayout>` component that has the switch between the normal layout and the modal.